easybuild/tools/repository/svnrepo.py

```diff
@@ -88,8 +88,8 @@ def setup_repo(self):
         try:
             pysvn.ClientError  # IGNORE:E0611 pysvn fails to recognize ClientError is available
         except NameError, err:
-            self.log.exception("pysvn not available (%s). Make sure it is installed " % err +
-                               "properly. Run 'python -c \"import pysvn\"' to test.")
+            self.log.exception("pysvn not available (%s). Make sure it is installed properly." 
+                                + " Run 'python -c \"import pysvn\"' to test.", err)
```

Comment:
this should be `log.error`?

also, I've found that the `, err` doesn't work, while `% err` does...

with `, err`, the string is not being formatted (or not always)